cmd/crypto/kes.go

```diff
@@ -115,15 +116,32 @@ func NewKes(cfg KesConfig) (KMS, error) {
 	if err != nil {
 		return nil, err
 	}
-	certPool, err := loadCACertificates(cfg.CAPath)
-	if err != nil {
-		return nil, err
-	}
-	cfg.Transport.TLSClientConfig = &tls.Config{
-		Certificates: []tls.Certificate{cert},
-		RootCAs:      certPool,
+	if cfg.Transport.TLSClientConfig != nil {
+		if err = loadCACertificates(cfg.CAPath,
+			cfg.Transport.TLSClientConfig.RootCAs); err != nil {
+			return nil, err
+		}
+		cfg.Transport.TLSClientConfig.Certificates = append(
+			cfg.Transport.TLSClientConfig.Certificates, cert)
```

Comment:
```suggestion
cfg.Transport.TLSClientConfig.Certificates = cert
```

KES requires that the client sends only one client certificate. Otherwise, KES cannot determine what policy to apply.